hw02-generics/src/main/java/homework/CustomerService.java

```diff
@@ -0,0 +1,43 @@
+package homework;
+
+
+import java.util.*;
+
+public class CustomerService {
+
+    TreeMap<Customer, String> customerList = new TreeMap<>();
```

Comment:
Обратите внимание, что мы сразу можем задать компаратор для TreeMap и хранить все в упорядоченном нам виде:
```suggestion
    private final TreeMap<Customer, String> customerList = new TreeMap<>(Comparator.comparingLong(Customer::getScores));
```

И дальше в `getSmallest()` нам нужен будет только `customerList.firstEntry()`.